query-planner-js/src/buildPlan.ts

```diff
@@ -2806,7 +2788,6 @@ export class QueryPlanner {
     // going to expand everything during the algorithm anyway. We'll re-optimize subgraph fetches with fragments
     // later if possible (which is why we saved them above before expansion).
     operation = operation.expandAllFragments();
```

Comment:
There is no reason to normalize at this point?

Reply:
Not now that `expandAllFragments` always ensure it's output is normalized.